virtwho/base.py

```diff
@@ -628,3 +629,25 @@ def system_reboot(ssh):
     if ssh_connect(ssh):
         return True
     raise FailException('Failed to reboot system')
+
+
+def json_data_create(hypervisors, guests):
```

Comment:
Used more time to understand this function and suggest to move the function to test_esx.py, because it only be used for esx mode and there is esx key word in `{"active": 1, "virtWhoType": "esx"}`, so it's not better to define in base.py.

Reply:
It's better to rename the param names to `def json_data_create(hypervisor_num, guest_num)`

Reply:
Okay